lib/src/comment_references/model_comment_reference.dart

```diff
@@ -123,3 +123,18 @@ class _ModelCommentReferenceImpl implements ModelCommentReference {
   late final List<CommentReferenceNode> parsed =
       CommentReferenceParser(codeRef).parse();
 }
+
+extension on CommentReferableExpression {
+  Element? get element {
+    var expression = this;
+    if (expression is PropertyAccess) {
```

Comment:
Might be better to sort these `if is X`.

Reply:
done